requirements/constraints.txt

```diff
@@ -16,8 +16,5 @@
 # the next major release, ensure the installed version is within boundaries.
```

Comment:
Could you add the link of the issue? Similar to [this](https://github.com/eduNEXT/eox-support/blob/5121a1611d7e99915786d20eda03d162581eda46/requirements/constraints.txt#L16).